R/getMetadataComment.R

```diff
@@ -0,0 +1,23 @@
+#' getMetadataComment
+#'
+#' Helper function extract a metadata comment
```

Comment:
```suggestion
#' Helper function to extract metadata from a magclass object's comment.
```

Reply:
actually it also works for other objects as we create a comment attribute also for non-magclass objects